lms/djangoapps/instructor/management/commands/collect_student_forums_data.py

```diff
@@ -0,0 +1,54 @@
+"""
+Command to retrieve aggregated student forums data in a .csv
+"""
+import csv
+import optparse
+import os
+
+from instructor.utils import collect_student_forums_data
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    """
+    Retrieve aggregated student forums data, write to .csv
+    """
+
+    help = ("Usage: collect_course_forums_data <course_id> --output-dir=<output_dir>")
+    args = "<course_id>"
+    option_list = BaseCommand.option_list + (
+        optparse.make_option('-o', '--output-dir',
```

Comment:
This block deviates from the community style guide [1].

[1] https://github.com/edx/edx-platform/wiki/Python-Guidelines